testdata/simple_expectation.json

```diff
@@ -7,7 +7,6 @@
     "properties": {
         "prop_bool": "true",
         "prop_empty_object": "{}",
-        "prop_empty_string": "",
```

Comment:
This test is impractical. Users should not really be setting property values to empty strings, they should define a property that accepts `null`. Also, our JQ expression runner has been modified to treat `""` as a jq expression that failed, so this is not a possible test case anymore.